testing/jormungandr-testing-utils/src/testing/vit/builder.rs

```diff
@@ -99,6 +112,11 @@ impl VotePlanBuilder {
         self
     }
 
+    pub fn voting_token(&mut self, voting_token: TokenIdentifier) -> &mut Self {
```

Comment:
isn't this the same as `with_voting_token`?

Reply:
Yes it is the same, I have done it with the respect of other fields. But yes, I agree with you it is better to leave only one method.